SWLOR.Game.Server/Service/AbilityService/RecastGroup.cs

```diff
@@ -25,14 +25,14 @@ public enum RecastGroup
         MindTrick = 8,
         [RecastGroup("Force Burst", "Force Burst", true)]
         ForceBurst = 9,
-        [RecastGroup("Force Body", "Force Body", true)]
-        ForceBody = 10,
+        [RecastGroup("Force Restore", "Force Restore", true)]
+        ForceRestore = 10,
```

Comment:
Changing the enumeration name on anything which is already stored in the database will cause errors when those players log into the server. You need to write a migration to do this rename on the raw player JSON files. In this instance, I would also be ok with just deleting the recast timer dictionaries on player records.

Reply:
Players don't really see recast timers so I can just change it back to force body and put force body/mind on force body's timer.

Otherwise I'll talk with you later to check on how to handle the migration stuff.